FAQ.md

```diff
@@ -2,13 +2,20 @@
 
 ## Resetting an accelerator board
 
-If a Tenstorrent chip seems to hang and/or is producing unexpected behaviour,
-you may try a software reset of the board.
+If you encounter a situation where a Tenstorrent chip appears to be unresponsive or is exhibiting unusual behavior, a software reset of the board might be a viable solution.
 
 For Grayskull: `tt-smi -tr all`
```

Comment:
Remove this line. It's redundant if you are mentioning `tt-smi -lr 0`.

Reply:
removing